test/test1.json

```diff
@@ -0,0 +1,184 @@
+        },
+        "title": {
+          "fieldName": "title",
+          "type": "String",
+          "description": "Title of the post",
+          "maxLength": null,
+          "default": null,
+          "nullable": false,
+          "unique": false,
+          "vectorEmbed": false,
+          "embeddingAlgo": ""
+        },
+        "content": {
+          "fieldName": "content",
+          "type": "String",
+          "description": "Content of the post",
+          "maxLength": null,
+          "default": null,
+          "nullable": false,
+          "unique": false,
+          "vectorEmbed": false,
+          "embeddingAlgo": ""
+        },
+        "author": {
+          "fieldName": "author",
+          "type": "User",
+          "description": "Author of the post",
+          "maxLength": null,
+          "default": null,
+          "nullable": false,
+          "unique": false,
+          "vectorEmbed": false,
+          "embeddingAlgo": ""
+        },
+        "comments": {
+          "fieldName": "comments",
+          "type": "Comment",
+          "description": "Comments associated with the post",
+          "maxLength": null,
+          "default": null,
+          "nullable": false,
+          "unique": false,
+          "vectorEmbed": false,
+          "embeddingAlgo": ""
+        }
+      },
+      "description": "Post model"
+    },
+    "Comment": {
+      "schemaName": "Comment",
+      "fields": {
+        "id": {
+          "fieldName": "id",
+          "type": "String",
+          "description": "ID of the comment",
+          "maxLength": null,
+          "default": null,
+          "nullable": false,
+          "unique": true,
+          "vectorEmbed": false,
+          "embeddingAlgo": ""
+        },
+        "content": {
+          "fieldName": "content",
+          "type": "String",
+          "description": "Content of the comment",
+          "maxLength": null,
+          "default": null,
+          "nullable": false,
+          "unique": false,
+          "vectorEmbed": false,
+          "embeddingAlgo": ""
+        },
+        "author": {
+          "fieldName": "author",
+          "type": "User",
+          "description": "Author of the comment",
+          "maxLength": null,
+          "default": null,
+          "nullable": false,
+          "unique": false,
+          "vectorEmbed": false,
+          "embeddingAlgo": ""
+        },
+        "post": {
+          "fieldName": "post",
+          "type": "Post",
+          "description": "Post associated with the comment",
+          "maxLength": null,
+          "default": null,
+          "nullable": false,
+          "unique": false,
+          "vectorEmbed": false,
+          "embeddingAlgo": ""
+        }
+      },
+      "description": "Comment model"
+    }
+  },
+  "description": "Prisma schema for blog application",
+  "dataSource": {
+    "name": "db",
+    "provider": "postgresql",
+    "url": {
+      "fromEnv": "DATABASE_URL"
+    }
```

Comment:
if user passes these options, then we should generate those otherwise, it should have a default

Reply:
Let's keep the default to be postgres only